<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>HistogramDialogWidget</class>
  <widget class="QWidget" name="HistogramDialogWidget" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>559</width>
        <height>565</height>
      </rect>
    </property>
    <layout class="QGridLayout" >
      <item row="0" column="0" >
        <widget class="Q3ButtonGroup" name="ButtonGroup1_3" >
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>5</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="title" >
            <string>Histogram Properties</string>
          </property>
          <layout class="QGridLayout" >
            <property name="margin" >
              <number>11</number>
            </property>
            <property name="spacing" >
              <number>6</number>
            </property>
            <item row="2" column="0" >
              <widget class="QLabel" name="TextLabel5_2" >
                <property name="sizePolicy" >
                  <sizepolicy>
                    <hsizetype>0</hsizetype>
                    <vsizetype>0</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="text" >
                  <string>Num bins:</string>
                </property>
                <property name="buddy" stdset="0" >
                  <cstring>N</cstring>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="1" column="4" >
              <widget class="QLabel" name="TextLabel4_5_2" >
                <property name="sizePolicy" >
                  <sizepolicy>
                    <hsizetype>0</hsizetype>
                    <vsizetype>0</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="text" >
                  <string>To:</string>
                </property>
                <property name="buddy" stdset="0" >
                  <cstring>Max</cstring>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item rowspan="1" row="0" column="0" colspan="2" >
              <widget class="QLabel" name="TextLabel1_10" >
                <property name="sizePolicy" >
                  <sizepolicy>
                    <hsizetype>0</hsizetype>
                    <vsizetype>0</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="text" >
                  <string>Data vector:</string>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item rowspan="1" row="0" column="2" colspan="4" >
              <widget class="VectorSelector" name="_vector" >
                <property name="sizePolicy" >
                  <sizepolicy>
                    <hsizetype>7</hsizetype>
                    <vsizetype>5</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
              </widget>
            </item>
            <item row="1" column="0" >
              <widget class="QLabel" name="TextLabel1_4_2_2" >
                <property name="sizePolicy" >
                  <sizepolicy>
                    <hsizetype>0</hsizetype>
                    <vsizetype>0</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="text" >
                  <string>From:</string>
                </property>
                <property name="buddy" stdset="0" >
                  <cstring>Min</cstring>
                </property>
                <property name="wordWrap" >
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item rowspan="1" row="1" column="1" colspan="3" >
              <widget class="QLineEdit" name="Min" >
                <property name="sizePolicy" >
                  <sizepolicy>
                    <hsizetype>7</hsizetype>
                    <vsizetype>0</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="text" >
                  <string>-10.0</string>
                </property>
                <property name="whatsThis" stdset="0" >
                  <string>The lower limit of the lowest bin.  Values below this limit will not be counted in any bin.</string>
                </property>
              </widget>
            </item>
            <item row="1" column="5" >
              <widget class="QLineEdit" name="Max" >
                <property name="sizePolicy" >
                  <sizepolicy>
                    <hsizetype>7</hsizetype>
                    <vsizetype>0</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="text" >
                  <string>10.0</string>
                </property>
                <property name="whatsThis" stdset="0" >
                  <string>The upper limit of the highest bin.  Values above this limit will not be counted in any bin.</string>
                </property>
              </widget>
            </item>
            <item rowspan="1" row="2" column="1" colspan="2" >
              <widget class="QSpinBox" name="N" >
                <property name="sizePolicy" >
                  <sizepolicy>
                    <hsizetype>7</hsizetype>
                    <vsizetype>0</vsizetype>
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="maximum" >
                  <number>1000000</number>
                </property>
                <property name="singleStep" >
                  <number>5</number>
                </property>
                <property name="value" >
                  <number>60</number>
                </property>
                <property name="whatsThis" stdset="0" >
                  <string>The number of bins in the Histogram.</string>
                </property>
              </widget>
            </item>
            <item rowspan="1" row="2" column="3" colspan="3" >
              <layout class="QHBoxLayout" >
                <property name="margin" >
                  <number>0</number>
                </property>
                <item>
                  <widget class="QPushButton" name="AutoBin" >
                    <property name="sizePolicy" >
                      <sizepolicy>
                        <hsizetype>0</hsizetype>
                        <vsizetype>0</vsizetype>
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                      </sizepolicy>
                    </property>
                    <property name="text" >
                      <string>A&amp;uto Bin</string>
                    </property>
                    <property name="whatsThis" stdset="0" >
                      <string>Set the upper and lower limits, and the number of bins to a reasonable default.</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QCheckBox" name="_realTimeAutoBin" >
                    <property name="text" >
                      <string>&amp;Real-time auto bin</string>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
          </layout>
        </widget>
      </item>
      <item row="2" column="0" >
        <widget class="CurveAppearanceWidget" name="_curveAppearance" />
      </item>
      <item row="1" column="0" >
        <widget class="Q3ButtonGroup" name="ButtonGroup5" >
          <property name="sizePolicy" >
            <sizepolicy>
              <hsizetype>5</hsizetype>
              <vsizetype>0</vsizetype>
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="title" >
            <string>Y Axis Normalization</string>
          </property>
          <layout class="QGridLayout" >
            <property name="margin" >
              <number>11</number>
            </property>
            <property name="spacing" >
              <number>6</number>
            </property>
            <item row="1" column="0" >
              <widget class="QRadioButton" name="NormIsPercent" >
                <property name="text" >
                  <string>&amp;Percent in bin</string>
                </property>
                <property name="whatsThis" stdset="0" >
                  <string>The Y axis of the histogram reports the percent of the samples in the bin.</string>
                </property>
              </widget>
            </item>
            <item row="0" column="1" >
              <widget class="QRadioButton" name="NormIsFraction" >
                <property name="text" >
                  <string>&amp;Fraction in bin</string>
                </property>
                <property name="whatsThis" stdset="0" >
                  <string>The Y axis of the histogram reports the fraction of samples in the bin.</string>
                </property>
              </widget>
            </item>
            <item row="0" column="0" >
              <widget class="QRadioButton" name="NormIsNumber" >
                <property name="text" >
                  <string>&amp;Number in bin</string>
                </property>
                <property name="checked" >
                  <bool>true</bool>
                </property>
                <property name="whatsThis" stdset="0" >
                  <string>The Y axis of the histogram reports the number of samples in the bin.</string>
                </property>
              </widget>
            </item>
            <item row="1" column="1" >
              <widget class="QRadioButton" name="PeakIs1" >
                <property name="text" >
                  <string>Peak bin = &amp;1.0</string>
                </property>
                <property name="shortcut" >
                  <string>Alt+1</string>
                </property>
                <property name="whatsThis" stdset="0" >
                  <string>The Y axis of the histogram is normalized so that the highest bin is 1.</string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item row="3" column="0" >
        <widget class="CurvePlacementWidget" name="_curvePlacement" />
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <tabstops>
    <tabstop>_vector</tabstop>
    <tabstop>Min</tabstop>
    <tabstop>Max</tabstop>
    <tabstop>N</tabstop>
    <tabstop>AutoBin</tabstop>
    <tabstop>_realTimeAutoBin</tabstop>
    <tabstop>NormIsNumber</tabstop>
    <tabstop>_curveAppearance</tabstop>
    <tabstop>_curvePlacement</tabstop>
  </tabstops>
</ui>
